### ISSUE TITLE
Thread 'screen' panicked when attempting to rename a floating panel

### ISSUE THREAD

**Issue by DannyBen** [zellij-org/zellij#1323]
**Basic information**

`zellij --version`: `0.28.1`
`tput lines`: `34`
`tput cols`: `137`
`uname -av` or `ver`(Windows): `Linux ubuntu-focal 5.4.0-107-generic #121-Ubuntu SMP Thu Mar 24 16:04:27 UTC 2022 x86_64 x86_64 x86_64 GNU/Linux`

**Further information**

I have seen many panic issues, so I hope this is not a duplicate. If it is, feel free to close it.

**Error**

```
  × Thread 'screen' panicked.
  ├─▶ Originating Thread(s)
  │     1. stdin_handler_thread: AcceptInput
  │     2. screen_thread: UpdatePaneName
  │
  ├─▶ At zellij-server/src/tab/mod.rs:1740:18
  ╰─▶ called `Option::unwrap()` on a `None` value
  help: If you are seeing this message, it means that something went wrong.
        Please report this error to the github issue.
        (https://github.com/zellij-org/zellij/issues)

        Also, if you want to see the backtrace, you can set the `RUST_BACKTRACE` environment variable to `1`.
```

then I rerun with backtrace on, hoping it would help the debugging effort, but it seems to not add any new infomration:

```
Error occurred in server:

  × Thread 'screen' panicked.
  ├─▶ Originating Thread(s)
  │     1. stdin_handler_thread: AcceptInput
  │     2. screen_thread: UpdatePaneName
  │
  ├─▶ At zellij-server/src/tab/mod.rs:1740:18
  ╰─▶ called `Option::unwrap()` on a `None` value
         0: <unknown>
        ..: <...trimmed - all unknown>
        13: <unknown>
```

**Reproduction steps**

1. Start zellij after a fresh install (no config)
2. Create a new floating pane <kbd>Ctrl</kbd>+<kbd>P</kbd> <kbd>W</kbd>
3. Rename the floating pane <kbd>Ctrl</kbd>+<kbd>P</kbd> <kbd>C</kbd>
4. Observe the panic

**Comment by raphCode** [comment]
Thanks for reporting this, server stability aka not-crashing-sessions a big concern for us! :)

I can reproduce this an Arch.

Link to crash location:
https://github.com/zellij-org/zellij/blob/v0.28.1/zellij-server/src/tab/mod.rs#L1735-L1749

**Comment by imsnif** [comment]
Yeah, I'd say that's the issue too. We need to also be looking in `self.floating_panes` if they are visible. There are examples in a few other places (eg. when resizing the active pane). Would you like to take a look at this, @raphCode ?
